config/m4/ib.m4

```diff
@@ -144,6 +144,11 @@ AS_IF([test "x$with_ib" == xyes],
                        MLX5_WQE_CTRL_SOLICITED],
                       [], [], [[#include <infiniband/mlx5_hw.h>]])
 
+       LIBMLX5_VER=`rpm -qf /usr/include/infiniband/mlx5_hw.h`
```

Comment:
let's make it happen in Makefile, not during configure. In case if some recompiles UCX with running ./autogen.sh+./configure

Reply:
i think it is overkill, because: 
1. rpm exists in rhel distros only, others have zipper/deb/.... package managers
2. makefile should take care for cases when ofed installed in non-standard location (--with-ofed=...)

but have no fear, maybe we should use ```$with_ofed/bin/ofed_info -s``` output to print mofed package version used during compile time?

Reply:
@miked-mellanox 
1. we need some way to check the version, if possible, on a **best-effort** basis. adding "deb" support is a good idea.
1. agree, let's add it
1. "ofed_info -s" would not work for inbox, and since ofed installs rpm/deb, checking rpm/deb should cover both ofed and inbox case

Reply:
@yosefe @miked-mellanox so you are suggesting to check the OS in the ucx makefile and according to it run rpm/deb/zipper query? Should ucx really be managing this?
how would that take care of a case where ```--with-ofed=...``` was used?

Reply:
$with_ofed is always set and points by default to /usr.
probably need external script and detect package manager and then extract package name from file name, something like this: 
https://unix.stackexchange.com/questions/46081/identifying-the-system-package-manager

Reply:
maybe add a check of rpm only for now and extend this to other package managers later on?
kind of looks like an overkill for an error message right now IMHO...